apps/src/code-studio/dancePartySongArtistTags.js

```diff
@@ -80,23 +80,26 @@ export const SongTitlesToArtistTwitterHandle = {
   cestlavie_khaled: 'Khaled_Officiel',
   occidentaliskarma_francescogabbani: 'FrankGabbani',
   isthatforme_anitta: 'Anitta',
-  showdaspoderosas_anitta: 'Anitta',
```

Comment:
These songs were eventually added to the song selection list so just moving to 2022 songs below.